microsoft-edge/toc.yml

```diff
@@ -1029,89 +1046,183 @@
               href: extensions-chromium/developer-guide/sidebar.md
               displayName: sidebar extensions, sidepanel, side panel  # alt lookup phrase
 
-            - name: Best Practices
-              href: extensions-chromium/developer-guide/best-practices.md
-              displayName: Best Practices for extensions  # full title
+#           - name: Microsoft Edge Add-ons API  # jump page linking to "All About APIs" and specifically "JavaScript APIs"
+#             href:
 
-        - name: Manifest
-          items:
-            - name: Overview and timelines for migrating to Manifest V3
-              href: extensions-chromium/developer-guide/manifest-v3.md
+            - name: Manifest files
+              items:
+                - name: Overview and timelines for migrating to Manifest V3  # overview of Manifest V3, and timeline for migrating
+                  href: extensions-chromium/developer-guide/manifest-v3.md
 
-            - name: Migrate an extension from Manifest V2 to V3
-              href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
+                - name: Migrate an extension from Manifest V2 to V3
+                  href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
+
+                - name: Manifest file format for extensions
+                  href: extensions-chromium/getting-started/manifest-format.md
+
+            - name: APIs for extensions
+              items:
+                - name: Microsoft Edge Add-ons API 101
+                  href: extensions-chromium/developer-guide/apis-intro.md
+
+                - name: JavaScript APIs
+                  items:
+                    - name: Supported APIs for Microsoft Edge extensions
+                      href: extensions-chromium/developer-guide/api-support.md
+
+                    - name: Declare API permissions in extension manifests
+                      href: extensions-chromium/developer-guide/declare-permissions.md
+
+                - name: REST API for publishing
```

Comment:
move REST API for publishing from the Develop (Build) bucket to the Publish bucket?
![image](https://github.com/user-attachments/assets/bc08107b-daa6-4b67-9673-c555c0fef30d)
@sivMSFT 
It's possible to have a 2nd representation of an article within the TOC, eg have a "jump page" within APIs bucket that contains just a link to the real article (REST APIs for publishing > "Using the Microsoft Edge Add-ons REST API") that's somewhere else in the toc

Reply:
i might make a commit that moves the REST articles into the Publishing bucket so that you can i can see what that looks like, vs the current placement that's in the inappropriate Develop bucket.
@sivMSFT

Reply:
moved "REST APIs" from Develop (Build) toc bucket to bottom of Publish bucket instead, b/c they are specifically for publishing, during publish phase, not during earlier Develop phase
@sivMSFT